packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -168,8 +175,20 @@ class AOTSnapshotter {
       ]);
     }
 
-    if (shouldStrip) {
-      genSnapshotArgs.add('--strip');
+    // When buiding for iOS and splitting out debug info, we want to strip
+    // manually after the dSYM export, instead of in the `gen_snapshot`.
+    late final bool stripAfterBuild;
```

Comment:
`late` not needed here.
```suggestion
    final bool stripAfterBuild;
```